packages/datetime/src/dateInput.tsx

```diff
@@ -72,9 +74,10 @@ export interface IDateInputProps extends IDatePickerBaseProps, IProps {
 
     /**
      * The format of the date. See http://momentjs.com/docs/#/displaying/format/.
+     * Alternatively pass an `IDateFormatter` for custom date rendering.
```

Comment:
Add comma after `Alternatively`:

```tsx
* Alternatively, pass an `IDateFormatter` for custom date rendering.
```

Also, no one is going to know what an `IDateFormatter` is. We should add docs on it.